Lib/functools.py

```diff
@@ -234,25 +234,26 @@ def __ge__(self, other):
 
 _initial_missing = object()
 
-def reduce(function, sequence, initial=_initial_missing):
+def reduce(function, iterable, initial=_initial_missing):
     """
-    reduce(function, sequence[, initial]) -> value
+    reduce(function, iterable[, initial]) -> value
 
-    Apply a function of two arguments cumulatively to the items of a sequence,
-    from left to right, so as to reduce the sequence to a single value.
+    Apply a function of two arguments cumulatively to the items of an iterable,
```

Comment:
Say, "sequence or iterable" here.

While *iterable* is technically correct, beginners find it to be less readable.

Reply:
I agree for this first appearance.